src/graphql/JobApplication/permissions.ts

```diff
@@ -0,0 +1,8 @@
+import { and } from "graphql-shield";
+import { isAuthenticated, isApplicant } from "../rules";
+
+export const jobApplicationPermissions = {
+  Mutation: {
+    saveJobApplication: and(isAuthenticated, isApplicant)
```

Comment:
hacer esto es como hacer and(isInteger, isPositiveInteger) en validations. uno incluye al otro conceptualmente, es un error permitir que se pueda poner isApplicant sin chequear tambien isAuthenticated